aimmdb/ingest/eli.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import pandas as pd
+from pathlib import Path
+
+
+def read_metadata_and_header(path: Path):
+    metadata = {}
+    header = ""
+    dat_file = path.open()
```

Comment:
Use a context manager here. It's much safer. For example, in this instance you forgot to close the file 😊

For example:

```python
with open(path, "r") as f:
    lines = f.readlines()
```